resources/lang/ep_gui.pt-BR.rc

```diff
@@ -424,7 +424,7 @@ BEGIN
     IDS_ABOUT_EXPORT_SUCCESS    "As configurações foram exportadas com sucesso."
     IDS_ABOUT_RESET             "Restaurar configurações padrão"
 
-    IDS_MAINTENANCE             "Configurações e desinstalação"
+    IDS_MAINTENANCE             "Configurações"
```

Comment:
I'd say keep the uninstall text here. Is it possible to appreviate it using a period like `Settings and Uninst.`?

Reply:
Done. I put "Desinst.", which is how we usually abbreviate "Desinstalação" (Uninstall) in pt-BR.